_config/products.yml

```diff
@@ -540,28 +551,6 @@ devstudio_is:
             file_size: 294MB
             url: https://devstudio.jboss.com/updates/8.0.0/jbdevstudio-integration-stack-updatesite-8.0.0.GA.zip
             md5_url: https://devstudio.jboss.com/updates/8.0.0/jbdevstudio-integration-stack-updatesite-8.0.0.GA.zip.MD5
-      8.0.0.Beta2:
```

Comment:
why are you removing these older entries ? We keep all previous versions, including Alpha/Beta/CR